src/Compilers/CSharp/Portable/Symbols/SubstitutedMethodSymbol.cs

```diff
@@ -386,8 +386,11 @@ private int ComputeHashCode()
 
         public override bool Equals(object obj)
```

Comment:
>public override bool Equals(object obj) [](start = 8, length = 39)

Can this override be removed?

Reply:
We're still using it in some places. This PR doesn't completely remove all our instances of internal equals. If we go that route, we can remove this and obsolete the others. If not we'll need to track it and do it in a future pass.

---
In reply to: [305450965](https://github.com/dotnet/roslyn/pull/37247#discussion_r305450965) [](ancestors = 305450965)